### PR TITLE
Adding a Python Course

### PR DESCRIPTION
**Pull request by Raffae2679**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by Raffae2679** [comment]
Why I get this error? Can someone help me? 
When I go to Travis CI to look what's the problem, It's say: 'The command "fpb-lint ." exited with 1."  
I don't know what's happening

File: free-courses-pt_BR.md [hunk 1]
```diff
@@ -202,13 +202,15 @@
...
 * [Introdução à linguagem Python](https://www.udemy.com/intro_python/) - Diego Mariano (Udemy)
+* [Programação em Python: O Guia para Iniciantes](https://www.udemy.com/course/programacao-python-guia-para-iniciantes/) - Jones Granatyr
 * [Python 3 na Prática](https://www.udemy.com/python-3-na-pratica/) - João Batista (Udemy)
 * [Python 3 na Web com Django (Básico e Intermediário)](https://www.udemy.com/python-3-na-web-com-django-basico-intermediario/) - Gileno Alves Santa Cruz Filho (Udemy)
 * [Python para Competições de Programação](https://www.youtube.com/playlist?list=PLMxflQ9_eOd9CY6Id5gfs3Edqt8vLC47p) - Adorilson (YouTube)
 * [Python para Iniciantes](https://www.udemy.com/python-para-iniciantes/) - Tiago Miguel (Udemy)
 * [Selenium com Python](https://www.youtube.com/playlist?list=PLOQgLBuj2-3LqnMYKZZgzeC7CKCPF375B) - Eduardo Mendes (YouTube)
 
 
+
```
Comment (SethiPandi) [inline free-courses-pt_BR.md:213]: Remove this line break.

**Review by SethiPandi — COMMENTED** [review]
You just had an extra line break, that's all!

Look a little higher in the Travis log and you'll see the cause!

**Comment by SethiPandi** [comment]
```
free-programming-books/free-courses-pt_BR.md

199:1-210:125 warning Incorrect number of blank lines between last section and next heading
```

**Pull request closed without merging**